projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php

```diff
@@ -273,3 +276,47 @@ function wpcom_show_admin_interface_notice() {
 	);
 }
 add_action( 'admin_notices', 'wpcom_show_admin_interface_notice' );
+
+/**
+ * Fetches the current assignment to the duplicate views experiment.
+ *
+ * @return string
+ */
+function wpcom_fetch_duplicate_views_assignment() {
```

Comment:
Couldn't we return a boolean that checks if it's enabled or not? There's no value in checking for a string.